app/controllers/partners/users_controller.rb

```diff
@@ -40,6 +41,10 @@ def create
   private
 
   def find_account
-    @account = @partner.providers.find(params[:provider_id])
+    @account = @partner.providers.find(permitted_params[:provider_id])
+  end
+
+  def permitted_params
+    params.permit!.to_h
```

Comment:
We should list all needed/accepted keys instead.

Reply:
@hallelujah changes done